src/index.html

```diff
@@ -1,13 +1,423 @@
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details1.jpg"
+                      alt="part1"
+                      class="parts__image"
+                    >
+                  </div>
+                </div>
+                <div
+                  class="grid__item grid__item--desktop-9-12 grid__item--tablet-5-6"
+                >
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details2.jpg"
+                      alt="part1"
+                      class="parts__image"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="parts__description">
+              <h3 class="parts__title">Auto Unlock</h3>
+              <p class="parts__paragraph">
+                The app senses when you're nearby to
+                unlock automatically. GPS tracking so you
+                know where your bike is and can track it anytime.
+              </p>
+            </div>
+          </article>
+
+          <article class="details__parts parts">
+            <div class="parts__images">
+              <div class="grid">
+                <div
+                  class="grid__item grid__item--desktop-1-8 grid__item--tablet-1-4"
+                >
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details3.jpg"
+                      alt="part1"
+                      class="parts__image"
+                    >
+                  </div>
+                </div>
+                <div
+                  class="grid__item grid__item--desktop-9-12 grid__item--tablet-5-6"
+                >
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details4.jpg"
+                      alt="part1"
+                      class="parts__image"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="parts__description">
+              <h3 class="parts__title">Range & Integrated lights</h3>
+              <p class="parts__paragraph">
+                The removable battery has up to 70km battery autonomy
+                and weighs only 2.4 kg. Lights integrated into the
+                frame give you always-on visibility day and night.
+              </p>
+            </div>
+          </article>
+
+          <article class="details__parts parts">
+            <div class="parts__images">
+              <div class="grid">
+                <div
+                  class="grid__item grid__item--desktop-1-4 grid__item--tablet-1-2"
+                >
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details5.jpg"
+                      alt="part1"
```

Comment:
The alt text for the image should be more descriptive, like 'Lightweight feature'